lib/ansible/modules/cloud/azure/azure_rm_aci.py

```diff
@@ -0,0 +1,405 @@
+        elif self.state == 'absent':
+            self.delete_aci()
+            self.log("ACI instance deleted")
+
+        return self.results
+
+    def create_update_aci(self):
+        '''
+        Creates or updates a container service with the specified configuration of orchestrator, masters, and agents.
+
+        :return: deserialized ACS instance state dictionary
+        '''
+        self.log("Creating / Updating the ACS instance {0}".format(self.name))
+
+        # self.log("orchestrator_profile : {0}".format(parameters.orchestrator_profile))
+        # self.log("service_principal_profile : {0}".format(parameters.service_principal_profile))
+        # self.log("linux_profile : {0}".format(parameters.linux_profile))
+        # self.log("ssh from yaml : {0}".format(results.get('linux_profile')[0]))
+        # self.log("ssh : {0}".format(parameters.linux_profile.ssh))
+        # self.log("master_profile : {0}".format(parameters.master_profile))
+        # self.log("agent_pool_profiles : {0}".format(parameters.agent_pool_profiles))
+        # self.log("vm_diagnostics : {0}".format(parameters.diagnostics_profile.vm_diagnostics))
+
+        parameters = ContainerGroup(self.location,
+                                    None,
+                                    [Container(self.name, self.image, ResourceRequirements(ResourceRequests(1.5, 1)))],
+                                    None,
+                                    None,
+                                    None,
+                                    self.os)
+
+        try:
+            client = ContainerInstanceManagementClient(self.azure_credentials, self.subscription_id)
+            response = client.container_groups.create_or_update(self.resource_group, self.group_name, parameters)
+        except CloudError as exc:
+            self.log('Error attempting to create the container instance.')
+            self.fail("Error creating the ACS instance: {0}".format(str(exc)))
+        return create_aci_dict(response)
+
+    def delete_aci(self):
+        '''
+        Deletes the specified container instance in the specified subscription and resource group.
+        The operation does not delete other resources created as part of creating a container service,
+        including storage accounts, VMs, and availability sets.
+        All the other resources created with the container service are part of the same resource group and can be deleted individually.
+
+        :return: True
+        '''
+        self.log("Deleting the ACI instance {0}".format(self.name))
+        try:
+            client = ContainerInstanceManagementClient(self.azure_credentials, self.subscription_id)
+            response = client.delete(self.resource_group, self.name)
+        except CloudError as e:
+            self.log('Error attempting to delete the ACS instance.')
+            self.fail("Error deleting the ACS instance: {0}".format(str(e)))
+
+        return True
+
+    def get_acs(self):
```

Comment:
After create the new instance, we should have a method to get the external ip address for visit it. So, can we expose the get_acs as?